python/langwatch/src/langwatch/telemetry/sampling.py

```
@@ -0,0 +1,38 @@
+import opentelemetry.trace as trace_api
+
+__all__ = ["trace", "get_current_trace", "get_current_span", "sampling_rate"]
```

Ruff (0.8.2)

Undefined name `trace` in `__all__`

(F822)